net/pfSense-pkg-frr/files/usr/local/pkg/frr/inc/frr_bgp.inc

```diff
@@ -609,15 +609,12 @@ function frr_bgp_generate_neighbors() {
 			$tconf .= "\n";
 		}
 
-		if (frr_validate_intrange($neighbor['bfd_multiplier'], 2, 255)) {
-			$nconf .= "  neighbor {$neighbor['peer']} bfd {$neighbor['bfd_multiplier']}";
-			if (frr_validate_intrange($neighbor['bfd_minrx'], 50, 60000)) {
-				$nconf .= " {$neighbor['bfd_minrx']}";
-				if (frr_validate_intrange($neighbor['bfd_mintx'], 50, 60000)) {
-					$nconf .= " {$neighbor['bfd_mintx']}";
-				}
+		if (!empty($neighbor['bfdenable'])) {
+			$bfd_peer = !empty($neighbor['bfd_peer']) && $neighbor['bfd_peer'] <> 'default' ? $neighbor['bfd_peer'] : $bfd_peer = $neighbor['peer'];
```

Comment:
This line is quite long and the ternary operator use is not properly isolated in parenthesis, which will cause an error on PHP 7.4.

Each section of the test must be in its own pair of parenthesis, plus the whole ternary operator and its parameters must also be in a set of parenthesis.

This is a case where it probably would be best to have it as a proper if statement instead of using the ternary operator, since the current format is more confusing and by the time you line wrap it properly, it isn't saving space.